salt/salt.spec

```diff
@@ -1377,7 +1395,12 @@ rm -f %{_localstatedir}/cache/salt/minion/thin/version
 
 %files -n python3-salt
 %defattr(-,root,root,-)
-%{python3_sitelib}/*
+%dir %{python3_sitelib}/salt
+%dir %{python3_sitelib}/salt-*.egg-info
+%exclude %dir %{python3_sitelib}/salt/tests
+%exclude %{python3_sitelib}/salt/tests/*
```

Comment:
We don't need to exclude these anymore I think.

Reply:
dropped